libs/documentation/src/lib/components/subheader/demos/basic-display/subheader-basic-display.component.scss

```diff
@@ -0,0 +1,8 @@
+@media screen and (min-width: 1024px) {
```

Comment:
Is the css here specific to the demo page or will the MFE's also need to add this style to properly display the subheaders?

Reply:
This css is only needed for the demo. The css is pulling browser viewport rather than parent width when determining what device the user is on for rendering. When MFEs implement the subheader, it will take up majority of the space, while the demo only gives the subheader ~900px. This was leading to components overlapping when they should not be because the viewport and actual width allow were substantially different.